mobile/src/screens/HomeScreen.tsx

```diff
@@ -1,85 +1,147 @@
-import React, { useContext, useEffect } from 'react';
-import { StyleSheet, Text, View, FlatList } from 'react-native';
+import React, { useContext, useEffect, useState } from 'react';
+import { StyleSheet, Text, View, ScrollView, KeyboardAvoidingView, Platform, Dimensions } from 'react-native';
+
 import { Svg, Circle } from 'react-native-svg';
 
-import { ColumnBookList } from '../components/ColumnBookList';
+import { BookFilter } from '../components/BookFilter';
+import { PaginatedBookList } from '../components/PaginatedBookList';
 import { HorizontalBookList } from '../components/HorizontalBookList';
+import { LoadingCircle } from '../components/LoadingCircle';
 import { BookContext } from '../context/BookContext';
 import { TextStyles } from '../styles/TextStyles';
 import { Colors } from '../styles/Colors';
 import { I18nContext } from '../context/I18nContext';
-import { LoadingCircle } from '../components/LoadingCircle';
+import { Language } from '../models/Languages';
+import { Book } from '../models/Book';
+
+// how many books to display per page (in the Paginated Booklist)
+const booksPerPage = 9;
```

Comment:
Let's just make this a constant in the PaginatedBookList file, rather than a prop. Also, when you move it, CONST_CASE it please

Reply:
I had the booksPerPage in HomeScreen bc the loading/no results view was also there, so I wanted to make sure the heights were the same with/without books so the screen doesn't get cut off.